src/models/wikicitations/__init__.py

```diff
@@ -51,11 +51,12 @@ def __convert_wcd_entity_id_to_item_entity__(self, entity_id: str) -> ItemEntity
         wbi = WikibaseIntegrator()
         return wbi.item.get(entity_id)
 
-    def __delete_all_page_items__(self):
+    def __delete_all_page_items__(self) -> None:
         """Get all items and delete them one by one"""
-        items = self.__extract_item_ids__(sparql_result=self.__get_all_page_items__())
-        if items is not None and len(items) > 0:
-            number_of_items = len(items)
+        items = list(
+            self.__extract_item_ids__(sparql_result=self.__get_all_page_items__())
+        )
+        if number_of_items := len(items) > 0:
```

Comment:
What is the minimum version of Python supported?  The `:=` syntax is only Python >= 3.8!

Reply:
Let's state 3.8 as minimum then 😀